recipes/maskslic/meta.yaml

```
@@ -0,0 +1,45 @@
+{% set name = "maskslic" %}
+{% set version = "0.3.4" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/maskslic-{{ version }}.tar.gz
+  sha256: 2bcfc5521ab6e1302f7c532606954ba9273b737cf0adc9e2eba8ebfdc2d22266
+
+build:
+  number: 0
+  script: {{ PYTHON }} -m pip install . -vv
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+    - {{ compiler('cxx') }}
+  host:
+    - cython
+    - pip
+    - python
+    - numpy >=1.7.2
+  run:
+    - cython
+    - numpy >=1.7.2
```

```suggestion
    - {{ pin_compatible('numpy') }}
```
See [here](https://conda-forge.org/docs/maintainer/knowledge_base.html#building-against-numpy)